src/test/smart-components/portfolio/portfolio-item-detail/__snapshots__/item-detail-info-bar.test.js.snap

```diff
@@ -60,7 +60,7 @@ exports[`<ItemDetailInfoBar /> should render correctly 1`] = `
             className="elipsis-text-overflow"
           >
             <span>
-              baz
+              quux
```

Comment:
Is this change tied to the `display_name` removal?

Reply:
yes - the test this snapshot is based on has the portfolio name=quux and display_name=baz - so the snapshot was correctly changed to use the name value